stable/xray/values.yaml

```diff
@@ -339,8 +345,6 @@ analysis:
   image:
     repository: docker.bintray.io/jfrog/xray-analysis
     # version:
-  updateStrategy: RollingUpdate
```

Comment:
@chukka  Why did you remove this?

Reply:
This is removed and added  here -  https://github.com/jfrog/charts/pull/924/files#diff-82f9672c8009df101fffb76468a8d55fR18 (This is cleanup from xray 2.x charts)